sources/packages/backend/libs/integrations/src/sfas-integration/sfas-files/sfas-application-disbursement.record.ts

```diff
@@ -0,0 +1,48 @@
+import { parseDate, parseInteger } from "./sfas-parse-utils";
+import { SFASRecordIdentification } from "./sfas-record-identification";
+
+/**
+ * This record contain data related to the actual funding disbursements records in SFAS.
+ */
+export class SFASApplicationDisbursementRecord extends SFASRecordIdentification {
+  constructor(line: string) {
+    super(line);
+  }
+  /**
+   * The unique key/number used in SFAS to identify this application (application.application_idx).
+   */
+  get applicationId(): number {
```

Comment:
Should the methods consuming `parseInteger` have their return types updated to indicate the possible `null` result now?